secboot/secboot_sb.go

```diff
@@ -367,6 +369,7 @@ func RenameKeys(node string, renames map[string]string) error {
 
 	// TODO:FDEM:FIX: listing keys, then modifying could be a TOCTOU issue.
 	// we expect here nothing else is messing with the key slots.
+	// XXX: include recovery key slots sbListLUKS2ContainerRecoveryKeyNames
```

Comment:
This function is for factory reset for Ubuntu Core. The recovery key gets deleted if I recall during factory reset. So that would be useless. Maybe we should rename this function `RenameKeysForFactoryReset`.